src/melee.cpp

```diff
@@ -1003,7 +1003,7 @@ float Character::get_dodge() const
     //Dodge decreases linearly to 0 when below 50% stamina.
     const float stamina_ratio = static_cast<float>( get_stamina() ) / get_stamina_max();
```

Comment:
```suggestion
    const float stamina_ratio = static_cast<float>( get_stamina() ) / get_stamina_max();
    if( stamina_ratio <= .1 ) {
        //Don't attempt to dodge if stamina too low, to avoid getting locked in
        return 0.0f;
    }
```